static/js/tools/timeline/chart.tsx

```diff
@@ -200,7 +200,20 @@ class Chart extends Component<ChartPropsType> {
       }
     }
     // use mprop as the ylabel
-    const ylabelText = mprop.charAt(0).toUpperCase() + mprop.slice(1);
+    let ylabelText = mprop.charAt(0).toUpperCase() + mprop.slice(1);
+
+    // TODO(shifucun):
+    // Pick unit from one place and one stat var. This assumes all the stat var
+    // observation have same unit, which is mostly wrong but not strictly true.
+    // To do this accurately need a bigger change that group the stat vars at
+    // an early stage.
```

Comment:
in the meantime, could we list all the available units? it'll make the issue obvious (or not, depending on the severity)

Reply:
Done

Reply:
Also added unit in tooltip